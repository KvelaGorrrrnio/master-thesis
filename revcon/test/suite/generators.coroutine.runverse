test/suite/generators.coroutine.revcon:
[store]
  hist: []
  main: Ã˜

[stack]
[spawn main main end () | unresume main (hist) | despawn main main begin ()] <|> []

