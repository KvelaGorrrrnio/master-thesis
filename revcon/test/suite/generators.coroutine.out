test/suite/generators.coroutine.revcon:
[store]
  hist: [1, 1, 1, 1, 1, 16, 9, 4, 1]
  main: Ã˜

[stack]
[spawn main main begin () | resume main (hist) | despawn main main end ()] <|> []

